### ISSUE TITLE
Missing translation in IPLB manager

### ISSUE THREAD

**Issue by miton18** [ovh-ux/ovh-manager-cloud#1164]
### Description

When you want to apply modification on a Load balancer, an error message appear, even if the update task is OK.

### Steps to Reproduce

1. Go to an IP Loadbalancer product
2. Edit his configuration (anything)
3. Apply the changes

**Expected behavior:**

No error message

**Actual behavior:**

An error message is shown
```
Une erreur est survenue lors de l'application de la configuration du load balancer : pascalprecht.translate.$translateSanitization: Error cannot interpolate parameter due recursive object
```

**Reproduces how often:**
Every time you apply a new change

**Comment by antleblanc** [comment]
@miton18 Thanks for reporting this issue.

Could you please tell me If you are able to reproduce this issue from the production environment?

Thanks:)

**Comment by miton18** [comment]
I actualy see this issue on
https://www.ovh.com/manager/cloud/index.html#/network/iplb/loadbalancer-xxx/configuration

**Comment by miton18** [comment]
up ?

**Comment by antleblanc** [comment]
Ticket reference has been created: MANAGER-1930

**Comment by miton18** [comment]
Any update ?

**Comment by miton18** [comment]
Up ?

**Comment by FredericEspiau** [comment]
I'm on it :)

**Comment by FredericEspiau** [comment]
Will be fixed by #1218

**Comment by antleblanc** [comment]
@miton18 it will be handled next week

**Comment by miton18** [comment]
Any news ?

**Comment by miton18** [comment]
@antleblanc ?

**Comment by FredericEspiau** [comment]
@miton18 we're having a few issues with translations right now, we'll come back to you as soon as it is fixed

**Comment by FredericEspiau** [comment]
@miton18 it should be fixed now

**Comment by miton18** [comment]
Seems not :-/

**Comment by FredericEspiau** [comment]
We have issues with the translations right now, we are trying to fix the issue

**Comment by cortexx** [comment]
> Seems not :-/

Indead, it's still not fixed.

**Comment by thcdrt** [comment]
Hello, unfortunately I just have the same issue. It's really disturbing.

**Comment by gclem** [comment]
Same

**Comment by miton18** [comment]
Any update ?

**Comment by antleblanc** [comment]
Hi @miton18

Sorry about the delay of the responsive time.

After digging into the translation issue that you've reported; I can answer you that all associated translation for the given key are there.

The associated key is `iplb_configuration_apply_error` and the GitHub search result across the repository
can be found here: https://github.com/ovh-ux/ovh-manager-cloud/search?l=JSON&q=iplb_configuration_apply_error 

I've also take a screenshot where I wasn't able to reproduce the given issue:

![screenshot-cloud-network-iplb-loadbalancer-configuration](https://user-images.githubusercontent.com/428384/53483279-746cf280-3a81-11e9-87a3-61720d1b1c4f.png)

```txt
Une erreur est survenue lors de l'application de la configuration du load balancer : pascalprecht.translate.$translateSanitization: Error cannot interpolate parameter due recursive object
```

Here is the related pull request that fix this issue.
ref: https://github.com/ovh-ux/ovh-manager-cloud/pull/1218

We've added the fix on the `v7.12.3` release and we are now on the latest version `v7.18.2`.

Could you confirm if you still have the described error?

Thanks!

**Comment by ganeshkumar1989** [comment]
Hey @miton18 ,

We are able to reproduce the error right now and we are working on a fix for the same. Expect to hear from us very soon. And sorry once again for the delay.

Thanks!

**Comment by antleblanc** [comment]
Fixed by #1355
